### PR TITLE
Fixed facet search

### PR DESCRIPTION
**Pull request by adityaanurag**
@piyuesh23

### CODE REVIEW

File: docroot/modules/contrib/custom_solr_search/custom_solr_search.module [hunk 1]
```diff
@@ -31,17 +31,11 @@ function custom_solr_search_theme($existing, $type, $theme, $path) {
   return array (
     'custom_solr_search_result' => array(
       'variables' => array(
-        'url' => NULL,
         'title' => NULL,
         'author' => NULL,
         'publishDate' => NULL,
         'publisher' => NULL,
         'topic' => NULL,
-        'docid' => NULL,
```
Comment (piyuesh23) [inline docroot/modules/contrib/custom_solr_search/custom_solr_search.module:40]: Don't we need docid here now? I believe this is the attribute being used in theme preprocessor to get node urls in the result listing.

File: docroot/modules/contrib/custom_solr_search/src/Controller/BasicSearch.php [hunk 2]
```diff
@@ -55,17 +54,11 @@ public function search($server = NULL, $keyword = NULL) {
         }
         $render['result'][] = array(
           '#theme' => 'custom_solr_search_result',
-          '#url' => isset($result->url[0]) ? $result->url[0] : '',
           '#title' => isset($title) ? $title : '',
           '#author' => isset($result->author) ? implode(', ', $result->author) : '',
           '#publishDate' => isset($result->publishDate) ? implode(', ', $result->publishDate) : '',
           '#publisher' => isset($result->publisher) ? implode(', ', $result->publisher) : '',
           '#topic' => isset($result->topic) ? implode(', ', $result->topic) : '',
-          '#docid' => isset($result->id) ? $result->id : '',
```
Comment (piyuesh23) [inline docroot/modules/contrib/custom_solr_search/src/Controller/BasicSearch.php:64]: Sure. We don't need this attribute?

File: docroot/modules/contrib/custom_solr_search/custom_solr_search.module (hunk 1 above)
Comment (adityaanurag) [inline docroot/modules/contrib/custom_solr_search/custom_solr_search.module:40]: No we don't need as it was for old add annotation link which we removed.

File: docroot/modules/contrib/custom_solr_search/src/Controller/BasicSearch.php (hunk 2 above)
Comment (adityaanurag) [inline docroot/modules/contrib/custom_solr_search/src/Controller/BasicSearch.php:64]: yes we don't need as it was for old add annotation link which we removed.

**Pull request merged**